yjit/src/core.rs

```diff
@@ -371,6 +371,33 @@ impl From<Opnd> for YARVOpnd {
     }
 }
 
+/// Maximum index of stack temps that could be in a register
+pub const MAX_LIVE_TEMPS: u8 = 8;
+
+/// Bitmap of which stack temps are in a register
+#[derive(Copy, Clone, Default, Eq, Hash, PartialEq, Debug)]
+pub struct LiveTemps(u8);
```

Comment:
Since we changed `MAX_LIVE_TEMPS` to `MAX_REG_TEMPS`, I changed all "live temps" to "reg temps" (register-allocated stack temps) instead https://github.com/Shopify/ruby/pull/511/commits/aa8887e951eb25709e2d1f15ba34598399df5808.